src/components-examples/angular/form-field/form-field-group/form-field-group-example.ts

```diff
@@ -10,15 +10,13 @@ import { UntypedFormBuilder, UntypedFormGroup, Validators } from '@angular/forms
   templateUrl: 'form-field-group-example.html',
 })
 export class FormFieldGroupExample {
-  formGroup: UntypedFormGroup;
-  lastSubmission?: { name: string; date: Date };
+  formGroup = this._formBuilder.group({
+    name: ['', [Validators.required, Validators.minLength(3)]],
+    date: [new Date(), Validators.required],
+  });
+  lastSubmission?: { name?: string | null; date?: Date | null };
```

Comment:
I am confused why here '| null' is necessary, as we do know, that the form doesn't contain a nullish form control

Reply:
It's because of the FormGroup's type which is `<{name: FormControl<string | null>, date: FormControl<Date | null>}>`
To avoid this one could use `{ initialValueIsDefault: true }` but I think that's not necessary.